src/act.other.cpp

```diff
@@ -4456,12 +4456,13 @@ ACMD(do_spray)
         return;
       }
       struct obj_data *paint = read_object(OBJ_GRAFFITI, VIRTUAL);
-      snprintf(buf, sizeof(buf), "a piece of graffiti that says \"%s^n\"", argument);
+      snprintf(buf, sizeof(buf), "^g   %s^n", argument);
       paint->restring = str_dup(buf);
-      snprintf(buf, sizeof(buf), "\"%s^g\" is sprayed here.", argument);
+      snprintf(buf, sizeof(buf), "^g   %s^n", argument);
       paint->graffiti = str_dup(buf);
       obj_to_room(paint, ch->in_room);
 
+      send_to_char("You spray graffiti into the room.\r\n", ch);
```

Comment:
A 'room' is an artificial concept that doesn't apply in a lot of situations, a more generic message like "You tag the area" might fit better here.